team/indexing-scripts/scrape_formatter.py

```diff
@@ -17,19 +17,19 @@
             df = pd.read_csv(f)
 
             # isolate rows that have 200 status
-            good_urls = df.loc[df['Status'].eq("200 no error")]
+            good_urls = df.loc[df['status'].eq("200 no error")]
 
             # dedup lines
-            deduped = good_urls.drop_duplicates(subset=["URL"], keep="first", inplace=False, ignore_index=False)
+            deduped = good_urls.drop_duplicates(subset=["url"], keep="first", inplace=False, ignore_index=False)
 
-            # write text file with only URLs from those rows. additional options added to remove extra leading/trailing quotes
-            deduped['URL'].to_csv('processed/' + filename.replace(".csv", "") + ".txt", index=False, header=False, quoting=csv.QUOTE_NONE, quotechar="",  escapechar="\\")
```

Comment:
nice, I remember you showing me the step where you had to strip quotes, now you don't!

Reply:
Still need to do that with that RSS feed, unfortunately. I wonder if we could just use this script to do that for us. not sure it would save time, but worth a try.